packages/documentation/src/stories/components/spinner/spinner.stories.ts

```diff
@@ -0,0 +1,96 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit/static-html.js';
+import { BADGE } from '../../../../.storybook/constants';
+
+const meta: Meta = {
+  title: 'Components/Spinner',
+  render,
+  decorators: [story => generateDecorators(story)],
+  parameters: {
+    badges: [BADGE.BETA],
+  },
+  args: {
+    size: 'null',
+  },
+  argTypes: {
+    size: {
+      name: 'Size',
+      description: "Sets the size of the component's appearance.",
+      control: {
+        type: 'select',
+        labels: {
+          'loader-12': '12',
+          'loader-16': '16',
+          'loader-24': '24',
+          'loader-32': '32',
+          'loader-40': '40',
+          'loader-48': '48',
+          'null': 'Default (56)',
+          'loader-80': '80',
+        },
+      },
+      options: [
+        'loader-12',
+        'loader-16',
+        'loader-24',
+        'loader-32',
+        'loader-40',
+        'loader-48',
+        'null',
+        'loader-80',
+      ],
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function generateDecorators(story: any) {
+  return html` <div class="m-5 overflow-hidden">${story()}</div> `;
+}
+
+function render(args: Args) {
+  const classes = ['loader', 'm-auto', args.size].filter(c => c && c !== 'null').join(' ');
+
+  return html` <div class="spinner-bg">
+    <div class="loading-modal">
+      <div class="${classes}" role="status">
```

Comment:
About using `role=status`.

We might want to discuss it, there is no clear recommendation. 
- [Bootstrap uses `role=status`.](https://getbootstrap.com/docs/5.2/components/spinners/#about)
- [Access42 recommends `role=alert` for spinner](https://access42.net/chargements-asynchrones-donnees-bonnes-pratiques-accessibilite-numerique/).

For me, it makes more sense to use the `role=status` as the loading aspect is important, but is not important enough to interrupt all other actions, but it might be different from the context.

Links
- [MDN `role=status`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role)
- [MDN `role=alert`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role)
- [MDN `role=progressbar`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role)

Reply:
I think going with the MDN recommendation is the right way for us, thanks for the explanation.